communities/src/tests.rs

```diff
@@ -26,6 +28,8 @@ use test_utils::{
 	*,
 };
 
+use encointer_primitives::balances::BalanceType;
```

Comment:
I guess this is the IDE's automatic import, which does not work as we want it to. :)

Reply:
no i think i made this import manually. what's wrong with it?

Reply:
That it enters the newline, and it does not group it with the rest

Reply:
oh i see :)